catalog/ui/catalog-ui-search/src/main/webapp/component/query-interactions/query-interactions.view.js

```diff
@@ -28,45 +28,53 @@ define([
     return Marionette.ItemView.extend(Decorators.decorate({
```

Comment:
Can you update this to use the original formatting, so the changes are more clear?  Reformatting should be done separately so we know there are no code changes involved.